workspaces/liferay-sample-workspace/client-extensions/liferay-sample-fds-cell-renderer/package.json

```diff
@@ -1,16 +1,26 @@
 {
 	"dependencies": {
-		"@liferay/dxp-7.4": "*",
-		"@liferay/js-api": "0.2.0-pre.0"
+		"@liferay/js-api": "^0.2.0-pre.0",
```

Comment:
I'm a bit afaid of this change without having any safety net (thinking of the linting tool that checks these version numbers against DXP versions).

However, since this stuff is inherently unstable, we may begin like this and see what happens.

I would say this won't fail too much until we have enough DXP releases with enough changes in React versions so as to make anything fail. In other words: even if your project uses -say- 16.15.0 and portal ships with 16.12.0, you won't probably find any problem unless you use a very specific API.